tests/TestCase/I18n/DateTest.php

```diff
@@ -81,7 +81,7 @@ public function testI18nFormat(): void
 
         $format = [IntlDateFormatter::NONE, IntlDateFormatter::SHORT];
         $result = $time->i18nFormat($format);
-        $expected = '12:00 AM';
+        $expected = '12:00 AM';
```

Comment:
This change only works if all of our local distros are caught up. We might need to support both in the tests.

Reply:
I was thinking of that as well. So the easiest solution would be to adjust the tests to use `str_starts_with` till the point where the NNBSP is present.

Reply:
We can replace unicode spaces with ascii spaces before comparing. We do similar changes in other icu tests.